examples/createLayer_noLegend.html

```diff
@@ -15,8 +15,8 @@
 
   </style>
 
-  <link rel="stylesheet" href="http://libs.cartocdn.com/cartodb.js/v3/3.15/themes/css/cartodb.css" />
-  <!-- 
+  <link rel="stylesheet" href="https://libs.cartocdn.com/cartodb.js/v3/3.15/themes/css/cartodb.css" />
+  <!--
   <link rel="stylesheet" href="https://cartodb-libs.global.ssl.fastly.net/cartodb.js/v3/3.15/themes/css/cartodb.css" />
```

Comment:
should we remove this commented one?

Reply:
It is a different URL so I think it's better to leave it as is, just in case someone wants to import from that URL or the other one is failing.